messenger.go

```diff
@@ -174,7 +169,7 @@ func (m *Messenger) ProfileByID(id int64, profileFields ...string) (Profile, err
 	}
 
 	if len(profileFields) == 0 {
```

Comment:
I don't think we need to throw an error here, since there are cases where you might want to retrieve a profile with no information: potentially you just want to see if the profile exists in the first place (in which case as long as there isn't an error you should be OK).